ialirt_data_access/cli.py

```diff
@@ -125,8 +127,19 @@ def _data_product_query_parser(args: argparse.Namespace):
     query_params = {k: v for k, v in query_params.items() if v is not None}
     try:
         query_results = ialirt_data_access.data_product_query(**query_params)
-        logger.info("Query results: %s", query_results)
-        print(query_results)
+        logger.info("Returned %d records", len(query_results))
+
+        downloads_dir = args.downloads_dir
+        if downloads_dir is None:
+            downloads_dir = Path.home() / "Downloads"
+        downloads_dir.mkdir(parents=True, exist_ok=True)
```

Comment:
Will this overwrite an existing directory with the same name as `downloads_dir`?